src/main/scala/nl/knaw/dans/easy/multideposit/DDM.scala

```diff
@@ -21,24 +21,22 @@ object DDM {
   val profileFields: Dictionary =
     Map("DC_TITLE" -> "dc:title",
       "DC_DESCRIPTION" -> "dcterms:description",
-      "DC_CREATOR" -> "dc:creator",
+      "DC_CREATOR" -> "dc:creator", // TODO verwijder
```

Comment:
I think we shouldn't add TODOs that cannot be executed right away by anyone encounters them. It is better to make a formal decision about removing these fields or not. So, please, remove the comment and bring this up at a development team meeting.